config/core/build-configs.yaml

```diff
@@ -685,9 +685,9 @@ build_configs:
               - 'allnoconfig'
               - 'allmodconfig'
 
-      # clang-10 is the current minium clang version for -next
-      clang-10:
-        build_environment: clang-10
+      # clang-11 is the current minium clang version for -next
```

Comment:
Actually this comment is out-of-date, as clang-10 (and maybe clang-11 soon) is the minimal version for mainline.